.github/workflows/config.yml

```diff
@@ -128,7 +128,28 @@ jobs:
         run : |
           if [ $PACKS_CHANGED ]; then
             BRANCH_NAME=${REF#refs/heads/}
-            # TODO - Upload to the artifacts server of your choice.
+
```

Comment:
1. Are we sure we don't want to simply point them here to the documentation?
2. If we're sure, let's add a preface here that explains this section, the details, and how should you use it (plus, the pros and cons of each option)